salt/modules/win_dism.py

```diff
@@ -511,6 +514,58 @@ def add_package(
     return __salt__["cmd.run_all"](cmd)
 
 
+def add_provisioned_package(package, image=None, restart=False):
+    """
+    Provision a package using DISM. A provisioned package will install for new
```

Comment:
Can you add a versionadded for 3007.0 for this new function?

Reply:
Done